### PR TITLE
Align 45 prisoner offender search

### PR DESCRIPTION
**Pull request by jamesgrant-moj**

### CODE REVIEW

File: server/data/prisonSearchClient.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import nock from 'nock'
+import config from '../config'
+import { PrisonerMockDataA, PrisonerMockDataB } from './localMockData/prisonSearch'
+import PrisonSearchClient from './prisonerSearchClient'
```
Comment (steverendell) [inline server/data/prisonSearchClient.test.ts:4]: seems a bit inconsistent with naming .. PrisonSearchClient, prisonerSearchClient, OFFENDER_SEARCH_API ..
Our repos tend to use _prisoner_Search

Comment (jamesgrant-moj) [reply server/data/prisonSearchClient.test.ts]: Thanks for taking a look @steverendell. I have made an update to the PR let me know if it is consistent enough.

File: server/config.ts [hunk 2]
```diff
@@ -74,6 +74,14 @@ export default {
       },
       agent: new AgentConfig(Number(get('PRISON_API_TIMEOUT_DEADLINE', 20000))),
     },
+    prisonerSearchApi: {
+      url: get('PRISONER_SEARCH_API_URL', 'http://localhost:8082', requiredInProduction),
+      timeout: {
+        response: Number(get('OFFENDER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
+        deadline: Number(get('OFFENDER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
+      },
+      agent: new AgentConfig(Number(get('OFFENDER_SEARCH_API_TIMEOUT_DEADLINE', 20000))),
```
Comment (steverendell) [inline server/config.ts:83]: you probably need to change these env var names as well as PRISONER_SEARCH_API_URL ?

File: server/data/index.ts [hunk 3]
```diff
@@ -18,6 +20,10 @@ export const dataAccess = () => ({
   hmppsAuthClient: new HmppsAuthClient(new TokenStore(createRedisClient({ legacyMode: false }))),
 })
 
+export const prisonSearchAccess = () => ({
```
Comment (steverendell) [inline server/data/index.ts:23]: could this be 'prisonerSearchAccess' ?

**Comment by jamesgrant-moj** [comment]
Thanks for taking a look @steverendell  I have updated the PR.

**Review by davidmiddletonmoj — APPROVED** [review]
LGTM

**Pull request merged**